app/helpers/va_facilities_helper.rb

```diff
@@ -1,4 +1,15 @@
 module VaFacilitiesHelper
+  def crh_created_practices_count(visn_id)
+    crh = ClinicalResourceHub.find_by(visn_id: visn_id)
+    @practices_created_by_crh = crh.get_crh_created_practices(crh.id, is_user_guest: is_user_a_guest?)
+    @practices_created_by_crh.count
```

Comment:
I think we can reduce this to just two lines, and remove the `@practices_created_by_crh`. Since we only need the count, we can remove line `5` and just do:
```
crh.get_crh_created_practices(crh.id, is_user_guest: is_user_a_guest?).count
```

Reply:
Yep - that works.